step-parent/pom.xml

```diff
@@ -222,8 +222,7 @@
 				<artifactId>maven-surefire-plugin</artifactId>
 				<configuration>
 					<systemPropertyVariables>
-						<!-- this file must be available in the classpath, it is currently located in the step-common resources -->
-						<logback.configurationFile>logback-for-test.xml</logback.configurationFile>
+						<logback.configurationFile>${basedir}/../parent/logback-maven.xml</logback.configurationFile>
```

Comment:
Be careful when merging master to your branch, you lost here again a change from master